src/gui/egui_ui/modal.rs

```diff
@@ -48,6 +49,9 @@ impl ModalDialog {
     pub fn about(&mut self) {
         self.payload = Some(ModalPayload::About);
     }
+    pub fn keybinds(&mut self){
```

Comment:
You should also run `cargo fmt` on your pull request to make sure the formatting stays consistent.